### PR TITLE
Actions in reference

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…avo-functions-overview**](https://docs-7j5w94r15.avo-dev.app/implementation/avo-functions-overview) | [**…s/managed-destinations**](https://docs-7j5w94r15.avo-dev.app/implementation/destinations/managed-destinations) |
|:-:|:-:|
|<a href="https://docs-7j5w94r15.avo-dev.app/implementation/avo-functions-overview">Screenshot of /implementation/avo-functions-overview</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/GhxfBOtV17OIQmaK5bfhCU3oslBRcxxTLnBZMgWJf10BC03TsThsSrDA718OR8rDDYmNPNIvl8tKI5pRxlanWQ.png">(view full size)</a> |<a href="https://docs-7j5w94r15.avo-dev.app/implementation/destinations/managed-destinations">Screenshot of /implementation/destinations/managed-destinations</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Da7AW7e_LjISUarqEhAvfrfE8ywX1nvaq1xnSheSk0g_5yq8zks93dz6Gm4gwK7tO4MkSOmCTC16tKOsSjVjZg.png">(view full size)</a> |



| [**…art-custom-destination**](https://docs-7j5w94r15.avo-dev.app/implementation/start-custom-destination) | [**…e/tracking-plan/events**](https://docs-7j5w94r15.avo-dev.app/workspace/tracking-plan/events) |
|:-:|:-:|
|<a href="https://docs-7j5w94r15.avo-dev.app/implementation/start-custom-destination">Screenshot of /implementation/start-custom-destination</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/xpAy340wNR403v6iNJhVRvjvGQLTRlqlh0dXs5FC9ZCkeMxOyOihg8eELV2ED-QzxyZx2hqeJFEO48ySRXScyA.png">(view full size)</a> |<a href="https://docs-7j5w94r15.avo-dev.app/workspace/tracking-plan/events">Screenshot of /workspace/tracking-plan/events</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/VWWTdDmLP8XguLOdNTa37ZkxYV_050hNjRwmtqliQNCJ9WGzCpD_FVuGS0iq4DlYd922cobvroxEFwbZ30aWsA.png">(view full size)</a> |







Commit 618d6b374e3b84cb94160e86080ab5f5970bb3d7 (https://docs-7j5w94r15.avo-dev.app).

File: pages/implementation/destinations/managed-destinations.mdx [hunk 1]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
   />
 </center>
 
-If you choose Avo to manage your destination, you'll still have to include the destination SDK in your application, but the initialization of the library and all event actions: Log Event, Update User Properties, Identify User, Unidentify User, Log Revenue, Log Page View, will be handled by the Avo generated code, so you don't need to think about that.
+If you choose Avo to manage your destination, you'll still have to include the destination SDK in your application, but the initialization of the library and all event actions: Log Event, Update User Properties, Identify User, Unidentify User, Log Revenue, Log Page View, Update Groups, will be handled by the Avo generated code, so you don't need to think about that.
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:20]:
```suggestion
If you choose Avo to manage your destination, you'll still have to include the destination SDK in your application, but the initialization of the library and all event actions: `Log Event`, `Update User Properties`, `Identify User`, `Unidentify User`, `Log Revenue`, `Log Page View`, `Update Groups`, will be handled by the Avo generated code, so you don't need to think about that.
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 2]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
 
 - Avo will initialize the analytics SDK with the API key you provided in the Avo UI corresponding to the specified environment when you initialize Avo
 - Avo will call the track method of the analytics SDK (e.g. `track(eventName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Event` action is called and will include the event and system properties in the call
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:23]:
```suggestion
- `Log Event`: Avo will call the track method of the analytics SDK (e.g. `track(eventName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Event` action is called and will include the event and system properties in the call
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 3]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
+- Avo will associate the track method with the specified group if the group is provided in the `Log Event` action
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:24]:
```suggestion
- `Log event with Group`: Avo will associate the track method with the specified group if the event is associated with a group in the `Log Event` action.
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 4]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
 - Avo will call the user properties method of the analytics SDK (e.g. `identify(userId, userProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Update User Properties` action is called and will include the user properties in the call
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:25]:
```suggestion
- `Identify User`: Avo will call the user properties method of the analytics SDK (e.g. `identify(userId, userProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Update User Properties` action is called and will include the user properties in the call.
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 5]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
 - Avo will call the identify method of the analytics SDK (e.g. `identify(userId)` in the Segment SDK) when an Avo Function corresponding to an event with `Identify User` action is called
 - Avo will call the unidentify method of the analytics SDK (e.g. `identify(null)` in the Segment SDK) when an Avo Function corresponding to an event with `Unidentify User` action is called
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:27]:
```suggestion
- `Unidentify User`: Avo will call the unidentify method of the analytics SDK (e.g. `identify(null)` in the Segment SDK) when an Avo Function corresponding to an event with `Unidentify User` action is called
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 6]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
 - Avo will call the revenue method of the analytics SDK (e.g. `track("revenue", {"revenue": amount})` in the Segment SDK) when an Avo Function corresponding to an event with `Log Revenue` action is called
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:28]:
```suggestion
- `Log Revenue`: Avo will call the revenue method of the analytics SDK (e.g. `track("revenue", {"revenue": amount})` in the Segment SDK) when an Avo Function corresponding to an event with `Log Revenue` action is called
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 7]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
 - Avo will call the page method of the analytics SDK (e.g. `page(pageName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Page View` action is called and will include the event and system properties in the call
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:29]:
```suggestion
- `Log Page View`: Avo will call the page method of the analytics SDK (e.g. `page(pageName, eventProperties)` in the Segment SDK) when an Avo Function corresponding to an event with `Log Page View` action is called and will include the event and system properties in the call
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 8]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
+- Avo will associate current user with the specified group when an Avo event with the 'Update Groups' action is triggered
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:30]:
```suggestion
- `Associate user with Group`: Avo will associate current user with the specified group when an Avo event with the 'Update Groups' action is triggered
```

File: pages/implementation/destinations/managed-destinations.mdx [hunk 9]
```diff
@@ -17,15 +17,20 @@ Avo can manage a limited set of Analytics destination, but you can use **any** e
...
+- Avo will update specified group metadata when an Avo event with the 'Update Groups' action is triggered
```
Comment (stefaniabje) [inline pages/implementation/destinations/managed-destinations.mdx:31]:
```suggestion
- `Update Groups Properties`: Avo will update specified group metadata when an Avo event with the 'Update Groups' action is triggered.
```

**Comment by stefaniabje** [comment]
Added instructions to manage Actions from the event modal + minor refactor of the tracking-plan/events#actions section

image

**Comment by stefaniabje** [comment]
Added reference to Actions resources in the Avo Functions Overview:

image

**Comment by stefaniabje** [comment]
Prefixed the list of actions in Managed Destinations with a reference to the action, for easier skimmability:

image

**Review by stefaniabje — APPROVED** [review]

**Pull request merged**